source/get-started/bootable-usb.rst

```diff
@@ -33,184 +22,61 @@ Burn the |CL| image onto a USB drive
-   file system checksum errors in it. If this happens, burn the image again,
-   ensuring all the USB drive partitions are unmounted first.
-
-.. _bootable-usb-mac:
-
-Create a bootable USB drive on macOS\*
-**************************************
-
-* Make sure you completed all `Prerequisites`_.
-
-* :ref:`verify-mac` on macOS.
-
-Burn the |CL| image onto a USB drive
-====================================
-
-.. caution::
-
-   Burning an image formats the USB drive and destroys all pre-existing
-   content.  Back up your data before proceeding.
-
-#. Open a Terminal window.
+#. Press :guilabel:`Select Image`.
 
 #. Change directory to where the image resides.
 
-#. Plug in a USB drive and get its identifier:
-
-   .. code-block:: bash
-
-      diskutil list
-
-   This lists available disks and their partitions, as shown in Figure 1.
-
-   .. figure:: /_figures/bootable-usb/bootable-usb-mac-01.png
-      :scale: 100 %
-      :alt: Get USB drive identifier
-
-      Figure 1: macOS - Get USB drive identifier
-
-#. Unmount the USB drive identified in the previous step. For example, to unmount /dev/disk2. The device name of the USB may vary.
+#. Open the Image.
 
-   .. code-block:: bash
+.. figure:: /_figures/bootable-usb/balenaEtcher_ImgaeSlect.PNG
 
-      diskutil umountDisk /dev/disk2
+#. Plug in the USB drive.
 
-#. Burn the image onto the drive using the :command:`dd` command.
-   This example uses `./`, your current directory, and it shows how to burn
-   an image onto `/dev/disk2`:
+#. Identify the USB drive using the :guilabel:`Change` under the middel icon.
+   .. note::
+      This shows all USB drives attached to the system.
 
-   .. code-block:: bash
+.. figure:: /_figures/bootable-usb/balenaEtcher_DriveSlect.PNG
 
-      sudo dd if=./clear-[version number]-live-[desktop | server].iso of=/dev/disk2 bs=4m
+#. Select the proper drive and press :guilabel:`Continue`.
 
-   To accelerate the imaging process, add an ‘r’ before the device identifier. Example: `sudo dd if=./clear-30800-live-server.iso of=/dev/rdisk2 bs=4m`.
+.. figure:: /_figures/bootable-usb/balenaEtcher_ReadyToFlash.PNG
 
-   Press :kbd:`<CTRL>-T` to check imaging progress.
+#. When ready Press the  :guilabel:`Flash!` Button.
 
-#. Eject the USB drive.
+#. Flashing starting.
 
-   .. code-block:: bash
+.. figure:: /_figures/bootable-usb/balenaEtcher_StartingToFlash.PNG
 
-      diskutil eject /dev/disk2
+#. Flashing in progress.
 
-.. _bootable-usb-windows:
+.. figure:: /_figures/bootable-usb/balenaEtcher_Flashing.PNG
 
-Create a bootable USB drive on Windows
-**************************************
+#. Flashing complete.
 
-* Make sure you completed all `Prerequisites`_.
+.. figure:: /_figures/bootable-usb/balenaEtcher_Done.PNG
 
-* :ref:`verify-windows` on Windows\* OS.
+   .. note::
+      The process can take more than a few minutes. When the process completes, close BalenaEtcher.
+      
 
-Burn the |CL| image onto a USB drive
+Ejecting the |CL| image USB drive
```

Comment:
This probably doesn't need to be its own section especially if Etcher automatically ejects after a successful flash (I think it does).